sick-fits/backend/.vscode/settings.json

```diff
@@ -12,9 +12,20 @@
   "[javascriptreact]": {
     "editor.formatOnSave": false
   },
+  "[typescript]": {
+    "editor.formatOnSave": false
+  },
+  "[typescriptreact]": {
+    "editor.formatOnSave": false
+  },
   "eslint.alwaysShowStatus": true,
   "editor.codeActionsOnSave": {
     "source.fixAll": true
   },
-  "prettier.disableLanguages": ["javascript", "javascriptreact"]
+  "prettier.disableLanguages": [
```

Comment:
Haven't finished the course, but I believe all we need here is `"typescript"`. Unless of course, we are in fact writing JSX at some point.